### PR TITLE
Add login node in spack wrfv3 tutorial example

### PR DESCRIPTION
**Pull request by alyssa-sm**
This PR updates the spack wrfv3 example to include a login node.
When manually testing `sbatch /opt/apps/wrfv3/submit_wrfv3.sh` on the login node, the job successfully completes and the following is in the directory. 
```
[spackwrfv3-login-login-001 test_run]$ ls
aerosol.formatted                 co2_trans                                  ozone.formatted        rsl.error.0005  rsl.error.0029  rsl.error.0053  rsl.out.0017  rsl.out.0041  tr49t67
aerosol_lat.formatted             coeff_p.asc                                ozone_lat.formatted    rsl.error.0006  rsl.error.0030  rsl.error.0054  rsl.out.0018  rsl.out.0042  tr49t85
aerosol_lon.formatted             coeff_q.asc                                ozone_plev.formatted   rsl.error.0007  rsl.error.0031  rsl.error.0055  rsl.out.0019  rsl.out.0043  tr67t85
aerosol_plev.formatted            constants.asc                              p3_lookup_table_1.dat  rsl.error.0008  rsl.error.0032  rsl.error.0056  rsl.out.0020  rsl.out.0044  TSK.png
bulkdens.asc_s_0_03_0_9           create_p3_lookupTable_1.f90                PH.png                 rsl.error.0009  rsl.error.0033  rsl.error.0057  rsl.out.0021  rsl.out.0045  TSLB.png
bulkradii.asc_s_0_03_0_9          diffwrf.py                                 QVAPOR.png             rsl.error.0010  rsl.error.0034  rsl.error.0058  rsl.out.0022  rsl.out.0046  U.png
CAM_ABS_DATA                      ETAMPNEW_DATA                              RAINC.png              rsl.error.0011  rsl.error.0035  rsl.error.0059  rsl.out.0023  rsl.out.0047  URBPARM.TBL
CAM_AEROPT_DATA                   ETAMPNEW_DATA_DBL                          RAINNC.png             rsl.error.0012  rsl.error.0036  rsl.out.0000    rsl.out.0024  rsl.out.0048  URBPARM_UZE.TBL
CAMtr_volume_mixing_ratio.A1B     ETAMPNEW_DATA.expanded_rain                README.namelist        rsl.error.0013  rsl.error.0037  rsl.out.0001    rsl.out.0025  rsl.out.0049  VEGPARM.TBL
CAMtr_volume_mixing_ratio.A2      ETAMPNEW_DATA.expanded_rain_DBL            README.rasm_diag       rsl.error.0014  rsl.error.0038  rsl.out.0002    rsl.out.0026  rsl.out.0050  V.png
CAMtr_volume_mixing_ratio.RCP4.5  GENPARM.TBL                                README.tslist          rsl.error.0015  rsl.error.0039  rsl.out.0003    rsl.out.0027  rsl.out.0051  wind-turbine-1.tbl
CAMtr_volume_mixing_ratio.RCP6    grib2map.tbl                               README.txt             rsl.error.0016  rsl.error.0040  rsl.out.0004    rsl.out.0028  rsl.out.0052  W.png
CAMtr_volume_mixing_ratio.RCP8.5  gribmap.txt                                real.exe               rsl.error.0017  rsl.error.0041  rsl.out.0005    rsl.out.0029  rsl.out.0053  wrfbdy_d01
capacity.asc                      hostfile                                   RRTM_DATA              rsl.error.0018  rsl.error.0042  rsl.out.0006    rsl.out.0030  rsl.out.0054  wrf.exe
CCN_ACTIVATE.BIN                  kernels.asc_s_0_03_0_9                     RRTM_DATA_DBL          rsl.error.0019  rsl.error.0043  rsl.out.0007    rsl.out.0031  rsl.out.0055  wrfout_d01_2001-10-24_12:00:00
CLM_ALB_ICE_DFS_DATA              kernels_z.asc                              RRTMG_LW_DATA          rsl.error.0020  rsl.error.0044  rsl.out.0008    rsl.out.0032  rsl.out.0056  wrfout_d01_2001-10-24_12:00:00-ORIG
CLM_ALB_ICE_DRC_DATA              LANDUSE.TBL                                RRTMG_LW_DATA_DBL      rsl.error.0021  rsl.error.0045  rsl.out.0009    rsl.out.0033  rsl.out.0057  wrfrst_d01_2001-10-24_09:00:00
CLM_ASM_ICE_DFS_DATA              masses.asc                                 RRTMG_SW_DATA          rsl.error.0022  rsl.error.0046  rsl.out.0010    rsl.out.0034  rsl.out.0058
CLM_ASM_ICE_DRC_DATA              MPTABLE.TBL                                RRTMG_SW_DATA_DBL      rsl.error.0023  rsl.error.0047  rsl.out.0011    rsl.out.0035  rsl.out.0059
CLM_DRDSDT0_DATA                  MU.png                                     rsl.error.0000         rsl.error.0024  rsl.error.0048  rsl.out.0012    rsl.out.0036  slurm-1.out
CLM_EXT_ICE_DFS_DATA              namelist.input                             rsl.error.0001         rsl.error.0025  rsl.error.0049  rsl.out.0013    rsl.out.0037  SOILPARM.TBL
CLM_EXT_ICE_DRC_DATA              namelist.input.backup.2024-02-14_02_49_07  rsl.error.0002         rsl.error.0026  rsl.error.0050  rsl.out.0014    rsl.out.0038  tc.exe
CLM_KAPPA_DATA                    namelist.output                            rsl.error.0003         rsl.error.0027  rsl.error.0051  rsl.out.0015    rsl.out.0039  termvels.asc
CLM_TAU_DATA                      ndown.exe                                  rsl.error.0004         rsl.error.0028  rsl.error.0052  rsl.out.0016    rsl.out.0040  T.png
```

### CODE REVIEW

File: docs/tutorials/wrfv3/spack-wrfv3.yaml [hunk 1]
```diff
@@ -156,13 +156,21 @@ deployment_groups:
     use: [compute_nodeset]
     settings:
       partition_name: compute
+      is_default: true
+
+  - id: slurm_login
```
Comment (harshthakkar01) [inline docs/tutorials/wrfv3/spack-wrfv3.yaml:161]: Lets move this one below controller.

**Comment by harshthakkar01** [comment]
I would add this to release candidate to make sure all spack examples have login node in main.

**Review by harshthakkar01 — APPROVED** [review]

**Comment by harshthakkar01** [comment]
This is not a hard requirement but we may consider adding this to RC.  @nick-stroud  wdyt ?

**Comment by nick-stroud** [comment]
Let's not try to add this to RC.

File: docs/tutorials/wrfv3/spack-wrfv3.yaml (hunk 1 above)
Comment (harshthakkar01) [reply docs/tutorials/wrfv3/spack-wrfv3.yaml:161]: Ignore.

**Pull request merged**